bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -175,6 +180,32 @@ def _setattr(self, name, val):
 
         return setattr(self, name, val)
 
+    def _run_rootlogon(self):
+        hasargv = hasattr(sys, 'argv')
+        # custom logon file (must be after creation of ROOT globals)
+        if hasargv and not '-n' in sys.argv and not self.PyConfig.DisableRootLogon:
+            rootlogon = os.path.expanduser('~/.rootlogon.py')
+            if os.path.exists(rootlogon):
+                # could also have used execfile, but import is likely to give fewer surprises
+                import imp
```

Comment:
`imp` is deprecated since py3.4: https://docs.python.org/3/library/imp.html

Reply:
Will fix that (old code), thanks!

Reply:
I had a closer look at this, if I want to do the same that is done here with `imp` (import a source file), I can only use `importlib` from Python3.5 up. In particular, I would use the API that this example illustrates:

https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly

Such API is not present in Python<3.5, so I would need to have conditional code depending on the Python version. See also:

https://docs.python.org/3/library/imp.html#imp.load_module